std/algorithm/searching.d

```diff
@@ -4035,6 +4036,132 @@ if (isInputRange!Range && !isInfinite!Range &&
     assert(maxElement(arr) == S(145));
 }
 
+/** Returns an array of the minimum and maximum element in `r`.
```

Comment:
I think this function should rather return a named tuple:
```d
Tuple!(ElementType!Range, "min", ElementType!Range "max")
```

Reply:
PR please!